### PR TITLE
[IMP] assets: remove css files split

### PR DESCRIPTION
**Pull request by ged-odoo**
Before this commit, our asset bundles had a special behaviour for
bundles of type css: because of an old limitation in IE9, css files had
to be split to make sure they had less than 4095 rules per stylesheets.

Since we no longer support IE9, this code can be removed, which should
slightly simplify the bundle code, and should slightly reduce the number
of requests required to properly load odoo.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by ged-odoo** [comment]
do not ask robodoo for merge, just comment on this PR if you want (I'll ask odo for confirmation)

File: odoo/addons/base/models/assetsbundle.py [hunk 1]
```diff
@@ -304,7 +293,7 @@ def save_attachment(self, type, content, inc=None):
             unique=self.version,
             extra='%s' % ('rtl/' if type == 'css' and self.user_direction == 'rtl' else ''),
             name=fname,
-            page='',  # included in fname
+            sep='',
```
Comment (odony) [inline odoo/addons/base/models/assetsbundle.py]:
```suggestion
            sep='', # included in fname
```

File: odoo/addons/base/models/assetsbundle.py [hunk 2]
```diff
@@ -199,31 +196,27 @@ def checksum(self):
         return hashlib.sha1(check.encode('utf-8')).hexdigest()
 
     def _get_asset_template_url(self):
-        return "/web/content/{id}-{unique}/{extra}{name}{page}{type}"  # name contains inc
+        return "/web/content/{id}-{unique}/{extra}{name}{sep}{type}"  # name contains inc
```
Comment (odony) [inline odoo/addons/base/models/assetsbundle.py]: Update/Remove comment?

File: odoo/addons/test_assetsbundle/tests/test_assetsbundle.py [hunk 3]
```diff
@@ -274,28 +228,19 @@ def test_11_paginated_css_content_invalidation(self):
         self.assertNotEquals(version0, version1)
 
         # check if the previous attachment are correctly cleaned
-        self.assertEquals(len(self._any_ira_for_bundle('css')), 4)
+        self.assertEquals(len(self._any_ira_for_bundle('css')), 1)
 
-    def test_12_paginated_css_debug(self):
+    def test_12__css_debug(self):
```
Comment (odony) [inline odoo/addons/test_assetsbundle/tests/test_assetsbundle.py]: possibly remove 1 underscore inside the method name

**Comment by aab-odoo** [comment]
robodoo r+

**Comment by ged-odoo** [comment]
robodoo retry

**Pull request closed without merging**